src/components/sbb-teaser-hero/sbb-teaser-hero.tsx

```diff
@@ -11,123 +16,64 @@ import tokens from '@sbb-esta/lyne-design-tokens/dist/js/sbb-tokens.json';
+   * If set, the link will be opened in a new window.
    */
-  @Prop() public buttonText!: string;
+  @Prop() public openInNewWindow?: boolean;
 
   /**
-   * Image source property for sbb-image. See sbb-image for additional info
+   * If `openInNewWindow` is set, you should provide according information
+   * which will be read aloud for screenreader users (e.g. "Link target will
+   * open in a new window").
    */
-  @Prop() public imageSrc!: string;
+  @Prop() public newWindowInfoText?: string;
 
-  /**
-   * Image loading property. See sbb-image for additional info
-   */
-  @Prop() public imageLoading?: InterfaceImageAttributes['loading'] = 'eager';
+  /** Teaser title text, visually hidden,  necessary for screenreaders */
+  @Prop() public accessibilityTitle!: string;
 
   /**
-   * Link to open if the teaser is clicked/pressed.
+   * Check if accessibilityTitle is provided since it is a required prop,
+   * otherwise throw an error.
    */
-  @Prop() public link!: string;
+  /* eslint-disable */
+  @Watch('accessibilityTitle')
+  validateAccessibilityTitle(newValue: string) {
+    const isBlank = typeof newValue !== 'string' || newValue === '';
+    if (isBlank) {
+      throw new Error('accessibilityTitle: required');
+    }
+  }
 
-  /**
-   * If set, the link will be opened in a new window.
-   */
-  @Prop() public openInNewWindow?: boolean;
+  public componentWillLoad(): void {
+    // Validate prop
+    this.validateAccessibilityTitle(this.accessibilityTitle);
+  }
 
   /**
-   * If `openInNewWindow` is set, you should provide according information
-   * which will be read aloud for screenreader users (e.g. "Link target will
-   * open in a new window").
+   * ----------------------------------------------------------------
    */
-  @Prop() public newWindowInfoText?: string;
 
   public render(): JSX.Element {
     const linkAttributes = {};
+    const ariaLabel = this.accessibilityTitle;
 
     if (this.openInNewWindow) {
       linkAttributes['rel'] = 'external noopener nofollow';
       linkAttributes['target'] = '_blank';
     }
 
     return (
-      <a class="teaser-hero" href={this.link} {...linkAttributes}>
-        <sbb-image
-          class="teaser-hero__image"
-          pictureSizesConfig={JSON.stringify(this._pictureSizesConfig)}
-          customFocalPoint={true}
-          hideFromScreenreader={true}
-          imageSrc={this.imageSrc}
-          loading={this.imageLoading}
-          lqip
-          performanceMark="teaser-hero"
-          variant="teaser-hero"
-        ></sbb-image>
-
-        <sbb-panel
-          class="teaser-hero__panel"
-          buttonText={this.buttonText}
-          text={this.text}
-        ></sbb-panel>
+      <a class="teaser-hero" href={this.link} aria-label={ariaLabel} {...linkAttributes}>
+        <sbb-title level="1" visually-hidden="true" text={ariaLabel} />
+        <div class="teaser-hero__panel">
+          <slot name="panel" />
+        </div>
```

Comment:
block elements are not allowed inside a link.
```suggestion
        <span class="teaser-hero__panel">
          <slot name="panel" />
        </span>
```